### ISSUE TITLE
Mumble floods .xsession-errors if open during a hibernate+restart

### ISSUE THREAD

**Issue by kevinlyles** [mumble-voip/mumble#1381]
Hibernating with Mumble running got me a 257GB .xsession-errors file by several hours after I resumed my session.  I can reproduce it reliably, and only see the issue when Mumble is running.

Steps to reproduce:
1. Run Mumble
2. Connect to a server (may be optional, but I was connected)
3. Run pm-hibernate (or probably your preferred method of hibernating would work)
4. Boot back up
It might help to do a tail -F on .xsession-errors before you hibernate, so you can see it in action.

I get this message over and over:

> ALSAAudioInput: No such device: No such device

I checked the growth rate of the file.  It's about 5.6 MB/second, or a gigabyte every 3 minutes:

```
$ ls -l .xsession-errors; sleep 60; ls -l .xsession-errors
-rw------- 1 me me 221375152 Sep  2 22:19 .xsession-errors
-rw------- 1 me me 576201490 Sep  2 22:20 .xsession-errors
```

I'm happy to run tests, try patches, etc. to help solve this -- just let me know what you need me to do.

**Comment by crknadle** [comment]
François Gannaz ran into this ~/.xsession-errors flooding bug when using the ALSA backend and reported it to Debian as #779836 with a link to the Github bug:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=779836

I had him try a package built with the current patch as-is -- that fixes the .xsession-errors flooding but runs into another problem with the Mumble client not having sound after the computer coming back from sleep, and then hanging when trying to change the settings.  He's currently using a systemd hook to kill Mumble on sleep events as a workaround.

**Comment by mkrautz** [comment]
cc https://github.com/mumble-voip/mumble/pull/1765

**Comment by mkrautz** [comment]
From my testing, I believe this was fixed by @bheart in b4d48ef43dc83ad924392527084af393ace1b7ba.

I tested on Arch + pm-suspend (not enough swap space to hibernate). It recovered. The log wasn't filled with junk, and I was told: "PCM suspended, trying to resume"...

Closing. Please re-open or create a new issue if this problem persists.
